src/app/charts/core/templates/_helpers.tpl

```diff
@@ -596,3 +596,9 @@
   {{- end -}}
   {{- get $masterCache $chartName | toYaml -}}
 {{- end -}}
+
+{{- define "arkcase.conf.extraSpringProfiles" -}}
+  {{- if .Values.global.conf.core.extraSpringProfiles -}}
+    {{- .Values.global.conf.core.extraSpringProfiles -}}
```

Comment:
Be careful when doing "long navigations" through a map. If any of the intermediate objects doesn't exist, or isn't a map, this will cause the template rendering to explode loudly and with often confusing error messages.

For instance, when looking for `.Values.global.conf.core.extraSpringProfiles`, only `.Values.global` is guaranteed to exist. Thus, if either `.Values.global.conf` or `.Values.global.conf.core` don't exist, the rendering will fail, and this is undesired behavior because we don't want users to have to add values in the `global` map unless they're necessary (i.e. confiugration-by-default).